### PR TITLE
[circle-eval-diff] Add MAPE metric printer

### PR DESCRIPTION
**Pull request by jinevening**
This adds MAPE metric printer.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

File: compiler/circle-eval-diff/src/MetricPrinter.cpp [hunk 1]
```diff
@@ -180,6 +180,111 @@ void MAEPrinter::dump(std::ostream &os) const
   }
 }
 
+// TODO Remove duplicate codes with MAEPrinter
```
Comment (seanshpark) [inline compiler/circle-eval-diff/src/MetricPrinter.cpp:183]: `MAEPrinter` -> `MAPEPrinter` ?

File: compiler/circle-eval-diff/include/CircleEvalDiff.h [hunk 2]
```diff
@@ -33,6 +33,7 @@ enum class Metric
 {
   Undefined, // For debugging
   MAE,
+  MAPE,
```
Comment (seanshpark) [inline compiler/circle-eval-diff/include/CircleEvalDiff.h:36]: (optioanl) maybe add `// Mean Absolute Percentage Error` ?

File: compiler/circle-eval-diff/src/MetricPrinter.cpp (hunk 1 above)
Comment (jinevening) [reply compiler/circle-eval-diff/src/MetricPrinter.cpp:183]: There is duplicate code in MAEPrinter, so I added this comment.

File: compiler/circle-eval-diff/include/CircleEvalDiff.h (hunk 2 above)
Comment (jinevening) [reply compiler/circle-eval-diff/include/CircleEvalDiff.h:36]: I'll add comments for both MAE and MAPE in another PR.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**